.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+---
+inherit_from: .rubocop-rails.yml
```

Comment:
remove inherit option

1) please add option - enable new cops
2) please add - rubocop-rails, rubocop-performance, rubocop-rspec and rubocop-rake to requirement list
3) Configure some cops if needed